api/v1beta1/aerospikecluster_types.go

```diff
@@ -315,6 +315,12 @@ type RackConfig struct {
 	// Racks is the list of all racks
 	// +nullable
 	Racks []Rack `json:"racks,omitempty"`
+
+	// RestartPercentage is the percentage of rack nodes that will be restarted simultaneously
+	RestartPercentage int `json:"restartPercentage,omitempty"`
+
+	// RestartNodesCount is number of nodes that will be restarted simultaneously
+	RestartNodesCount int `json:"restartNodesCount,omitempty"`
```

Comment:
Does it make sense to combine these two fields and handle both exact count and percentage?
K8s Deployment has similar field for RollingUpdate which can take both exact count or percentage and based on the dataType, further processing is done.

```
// The maximum number of pods that can be unavailable during the update.
	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
	// Absolute number is calculated from percentage by rounding down.
	// This can not be 0 if MaxSurge is 0.
	// Defaults to 25%.
	// Example: when this is set to 30%, the old ReplicaSet can be scaled down to 70% of desired pods
	// immediately when the rolling update starts. Once new pods are ready, old ReplicaSet
	// can be scaled down further, followed by scaling up the new ReplicaSet, ensuring
	// that the total number of pods available at all times during the update is at
	// least 70% of desired pods.
	// +optional
	MaxUnavailable *intstr.IntOrString `json:"maxUnavailable,omitempty" protobuf:"bytes,1,opt,name=maxUnavailable"`
```